macaroonbakery/httpbakery/client.py

```diff
@@ -2,31 +2,30 @@
 # Licensed under the LGPLv3, see LICENCE file for details.
 import base64
 import json
-import requests
-from six.moves.http_cookies import SimpleCookie
-from six.moves.urllib.parse import urljoin
+import logging
 
 import macaroonbakery as bakery
 import macaroonbakery.checkers as checkers
+import requests
 from macaroonbakery import utils
-from macaroonbakery.httpbakery.interactor import (
-    LegacyInteractor,
-    WEB_BROWSER_INTERACTION_KIND,
-)
-from macaroonbakery.httpbakery.error import (
-    DischargeError,
-    ERR_DISCHARGE_REQUIRED,
-    ERR_INTERACTION_REQUIRED,
-    Error,
-    InteractionError,
-    InteractionMethodNotFound,
-)
-from macaroonbakery.httpbakery.error import BAKERY_PROTOCOL_HEADER
 from macaroonbakery.httpbakery.browser import WebBrowserInteractor
+from macaroonbakery.httpbakery.error import (BAKERY_PROTOCOL_HEADER,
```

Comment:
This is about personal preference but I'd really prefer this style:
```
from x import (
    y,
    z,
)
```

Reply:
*sigh* I agree. I just used the default isort configuration so that I wouldn't have to care, but it seems I need to specify `--multi-line=3` to get reasonable behaviour. I'll leave it as is for the time being, and add a make target and sort all imports in another PR if that's OK.